.github/pull_request_template.md

```diff
@@ -0,0 +1,21 @@
+## Description
+
+## Related links
+
+**Parent Issue/PR:**
+
+- PR
+
+**Links to the definitions of the words added:**
+
+- Word
```

Comment:
I wanted to keep these list items empty but pre-commit kept complaining so I added single words to the items.